.prettierrc.json

```diff
@@ -2,5 +2,8 @@
   "printWidth": 120,
   "trailingComma": "all",
   "tabWidth": 2,
-  "semi": true
+  "semi": true,
+  "rules": {
+    "indent": ["error", 2]
```

Comment:
We want this in teh `.eslintrc.json`. The prettier already has the formatting with `"tabWidth": 2`

Reply:
When I added these to the linter it just broke everything, I fixed the issue with the color tho if you wanted to look through again